.github/workflows/ci.yml

```diff
@@ -15,23 +15,25 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js 14
       uses: actions/setup-node@v1
       with:
-        node-version: '14.1.0'
+        node-version: '14'
```

Comment:
I'm using '14' to use the latest version available. It should be safe and we don't need to manually update this value when a new version is released.